Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -477,6 +477,56 @@ package actor SkipUnless {
     }
   }
 
+  package static func canSwiftPMCompileForIOS(
+    file: StaticString = #filePath,
+    line: UInt = #line
+  ) async throws {
+    struct NoSwiftInToolchain: Error {}
+
+    return try await shared.skipUnlessSupported(file: file, line: line) {
+      #if os(macOS)
+      guard let swift = await ToolchainRegistry.forTesting.default?.swift else {
+        throw NoSwiftInToolchain()
+      }
+
+      let project = try await SwiftPMTestProject(
+        files: [
+          "Lib/MyFile.swift": """
+          public func foo() {}
+          """,
+        ],
+        manifest: """
+        let package = Package(
+          name: "MyLibrary",
+          targets: [
+           .target(name: "Lib"),
+          ]
+        )
+        """
+      )
```

Comment:
I think you should be able to reduce this a little bit further to

```suggestion
      let project = try await SwiftPMTestProject(
        files: [
          "MyFile.swift": """
          public func foo() {}
          """
      )
```